stan/math/prim/fun.hpp

```diff
@@ -305,6 +305,7 @@
 #include <stan/math/prim/fun/sub_row.hpp>
 #include <stan/math/prim/fun/subtract.hpp>
 #include <stan/math/prim/fun/sum.hpp>
+#include <stan/math/prim/fun/symmetrize.hpp>
```

Comment:
The new file needs to be added here and the old one (symmetrize.hpp) removed from prim/fun.hpp as well as the files/tests deleted. Then this is good to go.

Reply:
Yes, I'm on my mac and trying out github desktop. Typically I use tortoisegit on windows and when I brought commits over here it didn't update the files but added completely new ones! Anyway, should be fixed now.

Reply:
No worries. Thanks.